osf/models/registrations.py

```diff
@@ -556,6 +556,12 @@ class Meta:
         get_latest_by = 'created'
 
 
+def get_default_id():
+    from django.apps import apps
```

Comment:
Why is the use of `django.apps` beneficial/necessary compared to importing the `RegistrationProvider` model directly?

Reply:
Because this is techinically part of the migration, it could be run in the migration state before the app is loaded, using django/apps to import this avoids import errors in that case.